### PR TITLE
Mm/blogpost05202020

### PR DESCRIPTION
**Pull request by MissyM2**
Created blogpost for 05/20/2020.  This has been reviewed and shows properly on my local.

### CODE REVIEW

File: _posts/2020-05-20-writing-plain-english-content.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+**EDITED:**
+Submissions for the Clerk position in the department of record keeping are due by March 31st. Applications must include the following: a state-issued ID, a birth certificate and proof of residency. Individuals under under 18 years of age are ineligible for consideration. 
+
+
+## 2. Utilize White Space
+Once you have finished writing your content, step back and look at how the words appear on the page. Large paragraphs and long sentences create intimidating walls of text that discourage readers. However, shorter paragraphs, table of contents, titled subsections, and bulleted lists encourage readers. Use the example below as your guide, with the text on the right as the ideal example of accessible reader-friendly formatting.
+
+*(INSERT "F-SHAPED READING" IMAGE (300x300 px MAX) - https://drive.google.com/file/d/17XYwrWMryRE_QNHf3tSWmO0L7iy7ijqC/view?usp=sharing)*
```
Comment (meissadia) [inline _posts/2020-05-20-writing-plain-english-content.md:32]: This looks like it should be replaced with the actual image.

File: _posts/2020-05-20-writing-plain-english-content.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+ 
+To help ensure that your writing is clear, concise, and easy to understand, we’ve outlined the most universal elements of our content development approaches below.
+
+## 1. Be Short and to the point 
+People tend to skim and scan, rapidly searching for what they need when reading online. [Research shows](https://www.nngroup.com/articles/how-long-do-users-stay-on-web-pages/) that if a reader’s interest isn’t piqued within 10 seconds they won’t interact with your content. One way to overcome this is to utilize the [Nielsen Norman Group’s](https://www.nngroup.com/articles/how-users-read-on-the-web/) “inverted pyramid” structure. This organizational structure, utilized famously by newspapers, places the most important and eye-catching ideas first and auxiliary facts at the end. 
+
+*(INSERT "INVERTED PYRAMID" IMAGE (300x300 px MAX) - https://drive.google.com/file/d/1ltm5HCAGDne4y8TuCn9pAhJ4rAvaVjMj/view?usp=sharing)*
```
Comment (meissadia) [inline _posts/2020-05-20-writing-plain-english-content.md:19]: This looks like it should be replaced with the actual image.

File: _posts/2020-05-20-writing-plain-english-content.md (hunk 1 above)
Comment (meissadia) [inline _posts/2020-05-20-writing-plain-english-content.md:32]: This looks like it should be replaced with the actual image.

File: _posts/2020-05-20-writing-plain-english-content.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+The list of words and phrases below can safely be deleted from copy without affecting any significant meaning in your sentence. If you're looking for more, the [18F Content Guide](https://content-guide.18f.gov/plain-language/) has a more comprehensive list and many of these words are routinely identified as errors by grammatic proofreading tools, like Microsoft Word spell check and [Grammarly](https://www.grammarly.com/).
+
+* **Subjective Adjectives/Adverbs**
+Very, kind of, sort of, just, really, almost, a lot, always, obviously, undoubtedly
+
+* **Redundant Phrases**
+Absolutely necessary, advance planning, added bonus, blend together, completely eliminate, whether or not
+
+* **Words/phrases that convey no meaning** 
+Absolutely ( @_L88eO1tSeKJN3sFG95h9g The word "Absolutely" is mentioned in the line above as well -- can we pick a different word here or above? ), completely, entirely, basically, actually, in order to, as a matter of fact
```
Comment (meissadia) [inline _posts/2020-05-20-writing-plain-english-content.md:72]: `@_L88eO1tSeKJN3sFG95h9g` doesn't look like it belongs here.  I could be wrong.

File: _posts/2020-05-20-writing-plain-english-content.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+* **Write Short sentences**
+Avoid sentences that exceed 3 lines of text. Whenever possible break up sentences to make them easier to understand. 
+ 
+
+## 3. Use the Active Voice
+Using an active voice in your writing makes your sentences compelling, dynamic, and authoritative. Every sentence should feature a clear subject that is performing an obvious task to a receiving object. Whenever possible, avoid the passive voice, which occurs when the subject of the sentence passively receives an action, creating a much more ambiguous sentence. Any easy way to correct this mistake is to ask, who or what is performing the action, and then place them in the driver's seat. 
+
+**PASSIVE:** An error was made regarding your transaction. *(Who made the error?)*
+**ACTIVE:** We made an error regarding your transaction.
```
Comment (meissadia) [inline _posts/2020-05-20-writing-plain-english-content.md:53]: It looks like these should be on separate lines but are showing up on a single line.  Add two spaces at the end of the PASSIVE line to break them up.

File: _posts/2020-05-20-writing-plain-english-content.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
+**PASSIVE:** Without the password, access will not be granted. *(Who has the password? Who will get access?)*
+**ACTIVE:** If you don't have the password, your employees will not be granted access.
```
Comment (meissadia) [inline _posts/2020-05-20-writing-plain-english-content.md:56]: It looks like these should be on separate lines but are showing up on a single line.  Add two spaces at the end of the PASSIVE line to break them up.

File: _posts/2020-05-20-writing-plain-english-content.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title:  "Writing Plain English Content for a Great User Experience"
+short_title: "Digestible content" 
+date: 2020-05-20
+categories:
+featimg: "https://images.unsplash.com/photo-1505682634904-d7c8d95cdc50?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=3300&q=80"
```
Comment (meissadia) [inline _posts/2020-05-20-writing-plain-english-content.md:7]: I would recommend downloading all images and hosting them from the Raft repo, as opposed to linking from the external sources. That way they will always be available, even if removed from the external site. 

For larger images (500 Kb +), we can run them through https://tinyjpg.com to reduce the size/load times.

File: _posts/2020-05-20-writing-plain-english-content.md (hunk 1 above)
Comment (MissyM2) [reply _posts/2020-05-20-writing-plain-english-content.md:32]: I replaced it with a screenshot.

File: _posts/2020-05-20-writing-plain-english-content.md (hunk 2 above)
Comment (MissyM2) [reply _posts/2020-05-20-writing-plain-english-content.md:19]: I replaced it with a screenshot.

File: _posts/2020-05-20-writing-plain-english-content.md (hunk 3 above)
Comment (MissyM2) [reply _posts/2020-05-20-writing-plain-english-content.md:72]: I took it out.

File: _posts/2020-05-20-writing-plain-english-content.md (hunk 4 above)
Comment (MissyM2) [reply _posts/2020-05-20-writing-plain-english-content.md:53]: done.

File: _posts/2020-05-20-writing-plain-english-content.md (hunk 5 above)
Comment (MissyM2) [reply _posts/2020-05-20-writing-plain-english-content.md:56]: done.

File: _posts/2020-05-20-writing-plain-english-content.md (hunk 6 above)
Comment (MissyM2) [reply _posts/2020-05-20-writing-plain-english-content.md:7]: I did screenshots.  They are all around 200kb.

**Pull request merged**